### PR TITLE
Adding Users, Makers, and Integration Settings

### PR DESCRIPTION
**Pull request by tsmyre**
I'm committing these on behalf of Jojo. I made some edits, and Zach, if you could take a look at the Integration section, that would be super!

### CODE REVIEW

**Comment by zdavis** [comment]
Rebased on master.

File: docs/administering/configuring/integration_settings.md [hunk 1]
```diff
@@ -104,23 +80,12 @@ The Manifold API handles OAuth callbacks. For Google, the callback route is loca
...
-```
-# Google OAuth Integration
-MANIFOLD_SETTING_INTEGRATIONS_GOOGLE_OAUTH_CLIENT_ID=
-MANIFOLD_SETTING_SECRETS_GOOGLE_OAUTH_CLIENT_SECRET=
-```
+These settings can also be [managed from environment variables](/docs/administering/configuring/managing_settings#managing-settings-with-environment-variables).
 
 ## Google Analytics
 
 Manifold ships with its own [built-in analytics reporting tool](../../backend/analytics.md), independent of third party systems. However, if you would like to secure usage data from Google as well, you can set up an analytics property in the Google Console. That property will enable Google ti return usage information about your instance in the Google Analytics interface.
```
Comment (tsmyre) [inline docs/administering/configuring/integration_settings.md:87]: Looks like we have a misspelling here: `ti` instead of `to`.

Comment (zdavis) [reply docs/administering/configuring/integration_settings.md:87]: I can fix real quick.

Comment (tsmyre) [reply docs/administering/configuring/integration_settings.md]: Thanks!

**Pull request merged**